### PR TITLE
remove warnings

### PR DESCRIPTION
**Pull request by adrienpacifico**
Reduce number of warning related to returning  view vs a copy issues by using .loc
(http://pandas.pydata.org/pandas-docs/stable/indexing.html#returning-a-view-versus-a-copy)

Results looks ok and test_agregates seams to return the same values has before, but this might need to be checked/compared more deeply.

Step_02 imputation has not been changed.

### CODE REVIEW

File: openfisca_france_data/input_data_builders/build_openfisca_survey_data/step_04_famille.py [hunk 1]
```diff
@@ -553,7 +553,7 @@ def famille(temporary_store = None, year = None):
 
     famille['rang'] = famille.kid.astype('int')
     while any(famille[(famille.rang != 0)].duplicated(subset = ['rang', 'noifam'])):
-        famille["rang"][famille.rang != 0] += famille[famille.rang != 0].copy().duplicated(
```
Comment (benjello) [inline openfisca_france_data/input_data_builders/build_openfisca_survey_data/step_04_famille.py]: Need spaces around +=

File: openfisca_france_data/input_data_builders/build_openfisca_survey_data/step_04_famille.py [hunk 2]
```diff
@@ -553,7 +553,7 @@ def famille(temporary_store = None, year = None):
...
+        famille.rang.loc[famille.rang != 0]+= famille[famille.rang != 0].copy().duplicated(
```
Comment (benjello) [inline openfisca_france_data/input_data_builders/build_openfisca_survey_data/step_04_famille.py]: Need spaces around +=

**Comment by benjello** [comment]
@adrienpacifico fix the missing spaces and we are good to go

**Comment by adrienpacifico** [comment]
Is it ok now ?

**Pull request merged**